cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *
+
+[id="cnv-ansible-modules_{context}"]
+= Automating {ProductName} management tasks by using Red Hat Ansible Automation
```

Comment:
@ousleyp just a general comment (not for this PR) - this attribute name is a little confusing, since we have "product-title" for OpenShift and if I hadn't build it, I wouldn't have known that this was the CNV product name. I'd suggest to change this attribute name to CNVProductName or something, so that it's easier to understand from the source that it's CNV's name.

Reply:
@bergerhoffer Good call, I have wondered if this is confusing. I can bring it up in our CNV meeting this week.